### PR TITLE
Add Resource Audit Log entries

### PR DESCRIPTION
**Pull request by hardillb**
part of #2755 
## Description

Add the Audit Log formating

## Related Issue(s)

#2755 

## Checklist

 - [x] I have read the [contribution guidelines](https://github.com/FlowFuse/flowfuse/blob/main/CONTRIBUTING.md)
 - [ ] Suitable unit/system level tests have been added and they pass 
 - [ ] Documentation has been updated
    - [ ] Upgrade instructions
    - [ ] Configuration details
    - [ ] Concepts
 - [ ] Changes `flowforge.yml`?
    - [ ] Issue/PR raised on `FlowFuse/helm` to update ConfigMap Template
    - [ ] Issue/PR raised on `FlowFuse/CloudProject` to update values for Staging/Production

## Labels

 - [ ] Backport needed? -> add the `backport` label
 - [ ] Includes a DB migration? -> add the `area:migration` label

### CODE REVIEW

File: frontend/src/components/audit-log/AuditEntryVerbose.vue [hunk 1]
```diff
@@ -501,6 +501,16 @@
         <span>Nodes have been removed via the "Manage Palette" option inside Node-RED</span>
     </template>
 
+    <template v-else-if="entry.event === 'resource.cpu'">
+        <label>Instance High CPU usage</label>
+        <span>Instance has spent more than 5 minutes at more than 75% of CPU limit</span>
```
Comment (knolleary) [inline frontend/src/components/audit-log/AuditEntryVerbose.vue:506]: I'm slightly concerned about having these numbers hardcoded here. If we find we want to tune the values, we're then stuck in how it gets reported to the user.

Should the audit log event include this information and we use it to generate the full message?

Comment (hardillb) [reply frontend/src/components/audit-log/AuditEntryVerbose.vue]: Sorted, at the moment it defaults to minutes, do we want to use `moment.humanize`?

Comment (knolleary) [reply frontend/src/components/audit-log/AuditEntryVerbose.vue]: I think until we have real experience of the results of this, it's hard to say. This'll do for now.

File: frontend/src/components/audit-log/AuditEntryVerbose.vue [hunk 2]
```diff
@@ -501,6 +501,16 @@
         <span>Nodes have been removed via the "Manage Palette" option inside Node-RED</span>
     </template>
 
+    <template v-else-if="entry.event === 'resource.cpu'">
+        <label>Instance High CPU usage</label>
+        <span>Instance has spent more than {{ entry.body.interval / 60 }} minutes at more than {{ entry.body.threshold }}% of CPU limit</span>
```
Comment (knolleary) [inline frontend/src/components/audit-log/AuditEntryVerbose.vue:506]:
```suggestion
        <span>Instance has spent more than {{ Math.floor(entry.body.interval / 60) }} minutes at more than {{ entry.body.threshold }}% of CPU limit</span>
```

File: frontend/src/components/audit-log/AuditEntryVerbose.vue [hunk 3]
```diff
@@ -501,6 +501,16 @@
...
+    </template>
+
+    <template v-else-if="entry.event === 'resource.memory'">
+        <label>Instance High Memory usage</label>
+        <span>Instance has spent more than {{ entry.body.interval / 60 }} minutes at more than {{ entry.body.threshold }}% of Memory limit</span>
```
Comment (knolleary) [inline frontend/src/components/audit-log/AuditEntryVerbose.vue:511]:
```suggestion
        <span>Instance has spent more than {{ Math.floor(entry.body.interval / 60) }} minutes at more than {{ entry.body.threshold }}% of Memory limit</span>
```

**Review by knolleary — APPROVED** [review]
Approved with a minor tweak - want to avoid decimal places when presenting this value to the user

**Pull request merged**